pkg/doc/signature/suite/ed25519signature2018/public_key_verifier_test.go

```diff
@@ -26,11 +32,27 @@ func TestPublicKeyVerifier_Verify(t *testing.T) {
 	require.NoError(t, err)
 
 	pubKey := &verifier.PublicKey{
-		Type:  kms.ED25519,
-		Value: signer.PublicKey,
+		Type:  kmsapi.ED25519,
+		Value: signer.PublicKeyBytes(),
 	}
 	v := NewPublicKeyVerifier()
 
 	err = v.Verify(pubKey, msg, msgSig)
 	require.NoError(t, err)
 }
+
+func newCryptoSigner(keyType kmsapi.KeyType) (signature.Signer, error) {
+	p := mockkms.NewProviderForKMS(storage.NewMockStoreProvider(), &noop.NoLock{})
+	localKMS, err := localkms.New("local-lock://custom/master/key/", p)
```

Comment:
in [support_test.go](https://github.com/hyperledger/aries-framework-go/pull/1874/files#diff-bc435fee068227dfd806579fb2b37277R232) and [public_key_verifier_test.go](https://github.com/hyperledger/aries-framework-go/pull/1874/files#diff-cd51e51ac196475cba88ec3e5f04f3eeR161) further down, you have these two lines in a createKMS() call.. make it consistent here.

Reply:
They belong to the different packages, that's why I decided to duplicate the creation of KMS instance (several lines of code) in every package.